airflow/ui/src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import { hot } from 'react-hot-loader';
```

Comment:
`.ts` vs `.tsx` file extension: I've mostly seen .ts used -- the `x` suffix is to say it's got JSX templates in it, right?

Reply:
Correct. Might be good to capture that distinction in developer documentation.